en/getting-started.md

```diff
@@ -1,10 +1,106 @@
 
+After creating the basic information the addition of all other bibliographical details is often cumbersome and error-prone. To ease this task, JabRef allows for an automatic completion of the bibliographic information by looking up the data in public databases.
+To use this feature just click on the "Update with bibliographic information from the web" button in the editor:
+
+![Update information from web](.gitbook/assets/getting-started-entry-editor-update-from-web.png)
+
+{% hint style="info" %}
+The found information is most accurate if an identifier like a "DOI" or "ISBN" is maintained. If you already know such an unique identifier, this can also be already the starting point to create a new entry without manual entering any information by using the "create from ID" feature in the Create entry dialog. For more information see: [Collect](https://docs.jabref.org/collect) > ["Add entry using an ID"](https://docs.jabref.org/collect/add-entry-using-an-id)
+{% endhint %}
+
+If additional information is found you will be asked in a dialog which information should be taken over:
+
+![Merging the existing and the web information](.gitbook/assets/getting-started-merge-entries.png)
+
+## Adding a full text document
+
+Usually you also want to attach a reference to the full-text of a reference. For this, you can use the "file" field in the "General" tab.
+Here you can either attach a file manually, search for an already existing local file matching the bibtexkey pattern, or trying to automatically download a matching full text from the web.
+
+{% hint style="info" %}
+In order to use the automated feature, it is necessary to set-up a file directory first. To do so, please go to "Options" > "Preferences", go to "File" section and select there an existing folder as the "Main file directory":
+
+![Setting up the main file directory](.gitbook/assets/getting-started-preferences.png)
+{% endhint %}
+
+To test the automatic download of full texts you can click on the "Get full-text" icon next to the file field, or choose "Lookup" -> "Search full text documents online" from the menu. As soon as a full-text is found, the file will be stored in the local file directory and linked to the entry:
+
+![Finding a full-text document online](.gitbook/assets/getting-started-entry-editor-full-text.png)
+
+To open the downloaded full text you can click on "file" icon before the file name - or use the same icon in the entry table:
+![Opening the full-text](.gitbook/assets/getting-started-open-fulltext.png)
+
+## Finding more references in the web
+
+If you want to search for other references, it is also possible to directly trigger a search in many of the most common bibliographic databases. To start a search just us the "Web Search" feature of JabRef:
+First select one of the existing data sources, enter a search term and click on "search":
+
+The search results will be shown in an window where you can select all the search hits to be added to your library.
```

Comment:
```suggestion
The search results will be shown in an window where you can select all the search hits to be added to your library.
You also have the option to directly download the fulltext document (if avaiable and possible).
```

Reply:
That's not relevant for a getting started in my opinion

Reply:
I was referring to the dialog, that it has an option to download the ffull text .